src/AppInstallerRepositoryCore/CompositeSource.cpp

```diff
@@ -1365,6 +1365,8 @@ namespace AppInstaller::Repository
         }
     }
 
+    using namespace anon;

```

Comment:
curious why do we need to change the empty namespace?

Reply:
I've learned that anonymous namespace does not output type information into the PDB in a way that debuggers can find currently, making it much harder to debug.  With a named namespace, the derived type information is found and so the debugger can show the member fields of a derived type (vtable -> type mapping).  One can manually do all of this with enough effort, and maybe eventually the debuggers will support it directly, but at this time I see simply naming the namespace as the easiest answer.